README.md

```diff
@@ -18,21 +18,23 @@ Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a pa
 
 Orion Context Broker allows you to manage all the whole lifecycle of context information including updates, queries, registrations and subscriptions. It is an NGSI9/10 server implementation to manage context information and its availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
 
-If this is your first approach with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).
+If this is your first approach with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/manuals/quick_start_guide.md).
```

Comment:
Yes, you have commented the same in other documents. But I forgot ;)

Fixed in fdf0ebc